template/nuxt.config.js

```diff
@@ -50,8 +50,8 @@ module.exports = {
     /*
     ** Run ESLint on save
     */
-    extend (config, ctx) {
-      if (ctx.isDev && ctx.isClient) {
+    extend (config, {isDev}) {
```

Comment:
You destructure `isDev` but then use `process.isClient` and `process.server` later on. You should just desctructure all at once.

```js
extend (config, {isDev, isClient, isServer}) {
```

https://nuxtjs.org/api/configuration-build#extend